graylog2-server/src/main/java/org/graylog2/configuration/MongoDbConfiguration.java

```diff
@@ -25,12 +25,17 @@
 import com.mongodb.MongoClientURI;
 
 public class MongoDbConfiguration {
+    @Documentation("Increase this value according to the maximum connections your MongoDB server can handle from a single client " +
+            "if you encounter MongoDB connection problems.")
     @Parameter(value = "mongodb_max_connections", validator = PositiveIntegerValidator.class)
     private int maxConnections = 1000;
 
+    @Documentation("MongoDB connection string. See https://docs.mongodb.com/manual/reference/connection-string/ for details")
     @Parameter(value = "mongodb_uri", required = true, validator = StringNotBlankValidator.class)
     private String uri = "mongodb://localhost/graylog";
 
+    @Documentation("How many attempts should the search version probe run before giving up. " +
```

Comment:
"search version probe" doesn't seem to be the right wording, no?

Reply:
ah, good catch, it's mongo. :( 
I will add a fix for that

Reply:
Already fixed here: https://github.com/Graylog2/graylog2-server/pull/19115
Thanks!